lambdas/service/app.py

```diff
@@ -650,6 +652,27 @@ def validate_filters(filters):
             else:
                 raise BadRequestError(f'The relation in the `filters` parameter entry for `{facet}`'
                                       f' must be one of {valid_relations}')
+            if facet == 'organismAge':
+                validate_organism_age_filter(facet, value)
+
+
+def validate_organism_age_filter(facet, values):
+    for value in values:
+        try:
+            value, unit = value['value'], value['unit']
+        except KeyError:
+            raise BadRequestError(
+                msg=f'The `{facet}` filter parameter must specify a '
+                    f'dictionary with `value` and `unit` keys. For '
+                    f'filtering on the absence of `value` or `unit` use `None`.')
```

Comment:
```suggestion
                msg=f'A filter for `{facet}` must specify a '
                    f'dictionary with `value` and `unit` keys.')
```

We should explain the None thing in the spec, using a concrete example.